### DISCUSSION TITLE
2023a on MacOS has no import option for custom wbo files

### DISCUSSION THREAD

**Discussion by KE7** [cyberbotics/webots#6042] [Q&A]
**Describe the Bug**
Running WeBots 2023a on MacOS Ventura MacBook Pro M1. 

**Steps to Reproduce**
1. File -> New Project
2. Select the new/import node button
3. Trying to select any node type
4. There is no option to import only to add.

**Expected behavior**
I should be able to select a base mode like Robot and import a file for it.

**Screenshots**

**System**
 - Operating System: [macOS Ventura]
 - Graphics Card: [M1]

**Comment by ad-daniel** [comment]
wbo support has been dropped since R2022b, you should define the objects as PROTO instead
